shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1516,10 +1516,31 @@ public boolean mustLog() {
 
     @Data
     public static final class AuthTokenCheckFailed implements Reply, WireCommand {
+        public final static int ERROR_CODE_TOKEN_CHECK_FAILED = 0;
+        public final static int ERROR_CODE_TOKEN_EXPIRED = 1;
         final WireCommandType type = WireCommandType.AUTH_TOKEN_CHECK_FAILED;
         final long requestId;
         final String serverStackTrace;
 
+        // -1 indicates un-specified (for backward compatibility)
```

Comment:
Rather than being comments, and expressed as an interger. This should be an Enum where each value has an integer. Then the upper layers don't need to compare against those statics they can directly switch or do an IF on the values of the enum, and the encoding/decoding from an integer is contained within this class.

Reply:
Sure, that's a good idea. I've modified the code to use an enum in the auth token check failed wire command to represent error codes.